sensor/AdblockPlugin.js

```diff
@@ -89,27 +91,36 @@ class AdblockPlugin extends Sensor {
     }
 
     async getAdblockConfig() {
-      const json = await rclient.getAsync(configKey);
+      const result = {};
       try {
-        if (json == null) {
-          const result = {};
+        if (!platform.isAdblockCustomizedSupported()) {
+          result["ads"] = "off"
```

Comment:
should be on

Reply:
forget to change  it back after doing some test